### ISSUE TITLE
python2.7: command not found

### ISSUE THREAD

**Issue by zhenzhenyang-psu** [cschin/Peregrine#26]
hello Jason,
Sorry for not being able to use docker. This error message of mine might be an issue with conda.
So the error message from run-xxx.bash.stderr is as follows:
task.sh: line 18: python2.7: command not found

So under task.sh, there is a command which calls python2.7:
time python2.7 -m pypeflow.do_task  /public/home/yangzhzh/projects/0_aiden_lab/3_peregrine_test_run/2_assembly/wd/0-seqdb/task.json

It is interesting because on the pbs cluster, I installed Peregrine with conda, and the python version I used for conda is python3. So the default python is python3. Python2.7 is not installed on the cluster. Is there a way to get around this error?
Thanks,
Zhenzhen

**Comment by cschin** [comment]
yes. it is possible that your conda is python2.7. The Peregrine code is only work for python3. One way to get around this error is to use the docker build or you should start with a clean conda environment to see if it helps.

**Comment by zhenzhenyang-psu** [comment]
great! I will try it!

获取 Outlook for Android<https://aka.ms/ghei36>

________________________________
